pytorch_lightning/trainer/connectors/logger_connector/result.py

```diff
@@ -46,16 +47,21 @@ class MetricSource(LightningEnum):
 class _Sync:
     fn: Optional[Callable] = None
     should: bool = False
+    rank_zero_only: bool = False
```

Comment:
this would be confusing as our func has the same name

Reply:
Not sure to have a bette name.